### PR TITLE
frontend/fix/#1277/yatri-sathi-bug-fixes

### PR DESCRIPTION
**Pull request by MercyQueen**
## Type of Change

- [x] Bugfix
- [ ] New feature
- [ ] Enhancement
- [ ] Refactoring
- [ ] Dependency updates

### CODE REVIEW

**Review by witcher-shailesh — APPROVED** [review]

File: Frontend/ui-customer/src/Components/DriverInfoCard/View.purs [hunk 1]
```diff
@@ -1094,6 +1162,6 @@ getVehicleImage variant = do
   let url = getAssetStoreLink FunctionCall
   case variant of 
     "TAXI" -> "ic_yellow_ambassador," <> url <> "ic_yellow_ambassador.png"
-    "TAXI_PLUS" -> "ic_yellow_ambassador," <> url <> "ic_yellow_ambassador.png"
+    "TAXI_PLUS" -> "ic_white_taxi,," <> url <> "ic_white_taxi.png"
```
Comment (Rohit4997) [inline Frontend/ui-customer/src/Components/DriverInfoCard/View.purs:1165]: Remove extra , Else URL wil not work

File: Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/Controller.purs [hunk 2]
```diff
@@ -2282,7 +2285,7 @@ normalRideFlow  (RideBookingRes response) state = do
           , isSearchLocation = NoView
           }
         , data
-          { driverInfoCardState = getDriverInfo (RideBookingRes response) (state.data.currentSearchResultType == QUOTES)
+          { driverInfoCardState = getDriverInfo (fromMaybe "TAXI" state.data.specialZoneSelectedVariant )(RideBookingRes response) (state.data.currentSearchResultType == QUOTES)
```
Comment (Rohit4997) [inline Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/Controller.purs:2288]: Remove fromMaybe

File: Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/Controller.purs [hunk 3]
```diff
@@ -2296,7 +2299,7 @@ specialZoneRideFlow  (RideBookingRes response) state = do
           , isSearchLocation = NoView
           }
         , data
-          { driverInfoCardState = getDriverInfo (RideBookingRes response) (state.data.currentSearchResultType == QUOTES)
+          { driverInfoCardState = getDriverInfo (fromMaybe "TAXI" state.data.specialZoneSelectedVariant) (RideBookingRes response) (state.data.currentSearchResultType == QUOTES)
```
Comment (Rohit4997) [inline Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/Controller.purs:2302]: Remove fromMaybe

**Review by Rohit4997 — CHANGES_REQUESTED** [review]

File: Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/View.purs [hunk 4]
```diff
@@ -2358,8 +2361,16 @@ nearByPickUpPointsView state push =
     , width MATCH_PARENT
     , orientation VERTICAL
     , padding $ Padding 5 20 0 5
+<<<<<<< HEAD
     , visibility if state.props.defaultPickUpPoint /= "" then VISIBLE else GONE
     ](map (\item -> MenuButton.view (push <<< MenuButtonActionController) (menuButtonConfig state item)) state.data.nearByPickUpPoints)
+=======
```
Comment (Rohit4997) [inline Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/View.purs:2367]: @MercyQueen  You pushed with merge conflicts?

**Review by Rohit4997 — APPROVED** [review]

**Pull request merged**